Added new bits from Bionomia

dshorthouse opened the pull request:
Bionomia's search service was adjusted to include `uri` and `other_names` in response to what you were doing here. These are now used in the template.

dshorthouse committed:
Added new bits from Bionomia

Bionomia's search service was adjusted to include `uri` and `other_names` in response to what you were doing here. These are now used in the template.

nickynicolson commented on templates/bionomia.md:
```diff
@@ -1,20 +1,28 @@
 {{#if orcid}}ORCID: {{orcid}}{{/if}}
 {{#if wikidata}}Wikidata: {{wikidata}}{{/if}}
-aliases: ["{{{query}}}"]
+uri: {{uri}}
```
Please can I just check that `other_names` includes the search string passed to the search service? That was the content of the `query` variable, allowing aliases to more easily propose links.

nickynicolson replied:
On consideration I don't think bionomia should include the query string in its `other_names`: it is the responsibility of this template to include it (and maybe should be a configurable setting in the script options)

nickynicolson commented on templates/bionomia.md:
```diff
@@ -1,20 +1,28 @@
 {{#if wikidata}}Wikidata: {{wikidata}}{{/if}}
-aliases: ["{{{query}}}"]
+uri: {{uri}}
+aliases: [{{other_names}}]
```
Element values in `other_names` may include commas, which are the delimiters separating the different entries in the `aliases` list. So each of the values from `other_names` will need to be quoted: ie something like:
```
aliases: [{{#if other_names}}{{#each other_names}} "{{this}}",{{/each}}{{/if}} "{{query}}"]
```
This snippet also adds the query value as discussed above.

dshorthouse committed:
Include quotes around each alias in yaml block

Comment at https://github.com/echinopscis/echinopscis-sample-vault/pull/27/commits/0afd49a819c715b6af1351fe0db2ee1035e7a832#r950137338
